crates/generator/src/generator.rs

```diff
@@ -561,6 +561,34 @@ impl Generator {
         }
     }
 
+    /// Get backend name by script_hash
+    pub fn get_backend_type<S: State + CodeStore>(
```

Comment:
Do we really need this function?, why can't just use `manage.get_backend().map(|backend| backend.type)`?

Reply:
There is a `cloned` step in fn `load_backend`.
I think using `load_backend` directly is not efficient, so I created fn `get_backend_type`.

Reply:
This function increase the complexity, and do not actually improve the performance. If you really think the `cloned` affects performance, we should refactor the `get_backend` function, just copy the code to another function do not solve the problem.

I suggest we remove this function and use `get_backend` instead.

Reply:
This function increase the complexity, and do not actually improve the performance. If you really think the `cloned` affects performance, we should refactor the `get_backend` function, just copy the code to another function do not solve the problem.

I suggest we remove this function and use `get_backend` instead.